.buildkite/scripts/health-report-tests/bootstrap.py

```diff
@@ -0,0 +1,99 @@
+"""
+Health Report Integration test bootstrapper with Python script
+    - A script to resolve Logstash version if not provided
+    - Download LS docker image and spin up
+    - When tests finished, teardown the Logstash
+"""
+import os
+import subprocess
+import util
+import yaml
+
+
+class Bootstrap:
+    ELASTIC_STACK_VERSIONS_URL = "https://artifacts-api.elastic.co/v1/versions"
+
+    def __init__(self) -> None:
+        f"""
+        A constructor of the {Bootstrap}.
+        Returns:
+            Resolves Logstash branch considering provided LS_VERSION
+            Checks out git branch
+        """
+        logstash_version = os.environ.get("LS_VERSION")
+        if logstash_version is None:
+            # version is not specified, use the main branch, no need to git checkout
+            print(f"LS_VERSION is not specified, using main branch.")
```

Comment:
suggest to rename LS_VERSION to LS_BRANCH. It can't handle version, eg 8.13.4.
```bash
$ LS_VERSION="8.13.4" python3 .buildkite/scripts/health-report-tests/main.py
Starting Logstash Health Report Integration test.
Using specified branch: 8.13.4
error: pathspec '8.13.4' did not match any file(s) known to git
Traceback (most recent call last):
  File "/Users/kcheng/GitHub/kaisecheng/logstash/.buildkite/scripts/health-report-tests/main.py", line 74, in <module>
    main()
  File "/Users/kcheng/GitHub/kaisecheng/logstash/.buildkite/scripts/health-report-tests/main.py", line 42, in main
    with BootstrapContextManager() as bootstrap:
  File "/Users/kcheng/GitHub/kaisecheng/logstash/.buildkite/scripts/health-report-tests/main.py", line 20, in __enter__
    self.bootstrap = Bootstrap()
                     ^^^^^^^^^^^
  File "/Users/kcheng/GitHub/kaisecheng/logstash/.buildkite/scripts/health-report-tests/bootstrap.py", line 31, in __init__
    util.git_check_out_branch(logstash_version)
  File "/Users/kcheng/GitHub/kaisecheng/logstash/.buildkite/scripts/health-report-tests/util.py", line 17, in git_check_out_branch
    run_or_raise_error(["git", "checkout", branch_name],
  File "/Users/kcheng/GitHub/kaisecheng/logstash/.buildkite/scripts/health-report-tests/util.py", line 26, in run_or_raise_error
    raise Exception(f"{full_error_message}")
Exception: Error occurred while checking out the 8.13.4 branch
```

Reply:
I have tried LS_VERSION=8.15. It doesn't run the test. Maybe that branch has no python script
```bash
$ LS_VERSION="8.15" python3 .buildkite/scripts/health-report-tests/main.py
Starting Logstash Health Report Integration test.
Using specified branch: 8.15
Switched to branch '8.15'
logstash-integration-failure_injector successfully installed.
```

Reply:
Yup, thank you for suggestion, makes sense to change to branch concept. I have updated with [this commit](https://github.com/elastic/logstash/pull/16467/commits/5520880179741a31b19d38ae546342616d7ba2ba).